api/src/routes/mail_template.ts

```diff
@@ -0,0 +1,56 @@
+import { prisma } from "../prisma";
+import express from "express";
+import { CustomRequest, Routing } from "./routing";
+import { Auth } from "../auth/auth";
+import { Parser } from "../parser";
+
+export class MailTemplateRouting extends Routing {
+    @Auth.authorization({ superStudent: true })
+    async getAll(req: CustomRequest, res: express.Response) {
+        const result = await prisma.mailTemplate.findMany();
```

Comment:
Ik denk dat het hier wel handig zou zijn om te kunnen filteren op basis van `name` en `mail_subject`. Zo kunnen we in de frontend eventueel een filter toevoegen zodat superstudenten gerichter kunnen zoeken naar de mail template die ze nodig hebben.

Reply:
Dit zal idd handig zijn. In `mail_subject` kunnen er variabelen staan en het gaat lijken op `name`. Ik zou dus enkel filteren op `name`.

Reply:
Ik vermoed dat filteren op name inderdaad voldoende is!